lib/command/run.js

```diff
@@ -21,7 +21,10 @@ module.exports = function (test, options) {
     config = Config.append(JSON.parse(options.override));
   }
 
-  if (!fileExists(outputDir = path.join(testRoot, config.output))) {
+  if (config.output.startsWith('/') || config.output.startsWith('\\')) outputDir = config.output;
```

Comment:
windows? ;)
Please also move this part to `utils`, like `detectAbsolutePath`

Reply:
@DavertMik Yep, this checks absolute path for Windows and Unix systems. Will move this to utils